webportal/webapp/src/app/account/passwordreset/passwordreset.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-passwordreset',
+  templateUrl: './passwordreset.component.html',
+  styleUrls: ['./passwordreset.component.css']
+})
+
+export class PasswordResetComponent implements OnInit {
+  password: string;
+  password_confirm: string;
+  errorMessage = '';
+  passwordMatch = true;
+
+  constructor() { }
+
+  ngOnInit() {
+  }
+
+  onSubmit() {
+    if(this.password != this.password_confirm) {
```

Comment:
Looking at our ADFS, the password requirements are:

- Minimum 7 characters

- 3 selections in: numbers, uppercase letters, lowercase letters, and special characters.

Can you add those as well?

-